src/Link/index.tsx

```diff
@@ -143,22 +151,54 @@ export default class Link extends React.PureComponent<LinkProps, LinkState> {
     this.props.onMouseOut(this.props.linkData.source, this.props.linkData.target, evt);
   };
 
+  getLinkColor(): string[] | string | undefined| null  {
+    const { linkData, pathColorFunc } = this.props;
+    let color = pathColorFunc;
+    if (typeof pathColorFunc === 'function') {
```

Comment:
Let's remove the support for non-function here - just always call `pathColorFunc` (also the Props typing says its always a function)

Reply:
We still need to check if pathColorFunc is a function

Reply:
Why? In TypeScript we shouldn't verify, we depend on the using code to have TypeScript as well, or to be responsible. See other function props above - they don't verify them.